lang/cs/Org.Apache.REEF.Client/YARN/IYarnREEFClient.cs

```diff
@@ -33,5 +34,13 @@ public interface IYarnREEFClient : IREEFClient
         /// <returns></returns>
         [Unstable("0.17", "Working in progress for rest API id returned")]
         Task<IReadOnlyDictionary<string, IApplicationReport>> GetApplicationReports();
+
+        /// <summary>
+        /// Kills the job application and return Job status
+        /// </summary>
+        /// <param name="appId"></param>
+        /// <returns></returns>
+        [Unstable("0.17", "Working in progress for rest API status returned")]
+        Task<FinalState> KillJobApplication(string appId);
```

Comment:
How does this behave if `appId` is invalid or refers to a job that never existed or has already ended?